meta-luneos/recipes-multimedia/alsa/alsa-ucm-conf/0005-Add-UCM-for-PinePhonePro.patch

```diff
@@ -619,3 +621,14 @@ index 0000000..2e8ffb4
 @@ -0,0 +1 @@
 +../../Rockchip/rk3399/PinePhonePro/PinePhonePro.conf
 \ No newline at end of file
+diff --git a/ucm2/conf.d/simple-card/rockchip,rt5640-codec.conf b/ucm2/conf.d/simple-card/rockchip,rt5640-codec.conf
+new file mode 100644
+index 0000000..6d1864f
+--- /dev/null
++++ b/ucm2/conf.d/simple-card/rockchip,rt5640-codec.conf
+@@ -0,0 +1 @@
++../../Rockchip/es8316/es8316.conf
```

Comment:
is that a symlink file ? the content looks a bit short

Reply:
> is that a symlink file ? the content looks a bit short

Yes took it like that from another repo. Think SFOS don't be evil's